src/Magnum/ImGuiIntegration/ImGui.vert

```diff
@@ -0,0 +1,61 @@
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015, 2016, 2017, 2018
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2018 Jonathan Hale <squareys@googlemail.com>
```

Comment:
Weren't the shaders also part of the original repositories? If so, would be nice to mention the corresponding people here to avoid frowny faces ;)